src/Microsoft.ML.FastTree/RandomForestClassification.cs

```diff
@@ -113,12 +113,54 @@ private static IPredictorProducing<float> Create(IHostEnvironment env, ModelLoad
     /// <summary>
     /// The <see cref="IEstimator{TTransformer}"/> for training a decision tree binary classification model using Fast Forest.
     /// </summary>
-    /// <include file='doc.xml' path='doc/members/member[@name="FastForest_remarks"]/*' />
```

Comment:
>FastForest_remarks [](start = 64, length = 18)

this content is share between FastForestBinary and FastForestRegression. please move it to ~\docs\api-reference\algo-details-fastforest.md and reference it with !include[algo]... similar to input/output

move the heading '### Training Algorithm Details ' inside the file.